src/server_manager/web_app/app.ts

```diff
@@ -247,6 +247,10 @@ export class App {
       }
     });
 
+    appRoot.addEventListener('ChangeLanguage', (event: CustomEvent) => {
```

Comment:
Maybe name the event `SetLanguageRequested`? The language hasn't changes yet.

Reply:
Done